crates/tauri-utils/src/platform.rs

```diff
@@ -264,7 +264,12 @@ fn is_cargo_output_directory(path: &std::path::Path) -> bool {
 /// On iOS, it's `${exe_dir}/assets`.
 ///
 /// Android uses a special URI prefix that is resolved by the Tauri file system plugin `asset://localhost/`
+///
+/// Additionally, the resource directory can be overriden by setting the `TAURI_RESOURCE_DIR` environment variable at compile time.
 pub fn resource_dir(package_info: &PackageInfo, env: &Env) -> crate::Result<PathBuf> {
+  if let Some(dir) = option_env!("TAURI_RESOURCE_DIR") {
```

Comment:
this is not a really portable solution since you'd need separate builds per target platform; following the explanation from https://github.com/tauri-apps/tauri/issues/8679#issuecomment-1911288087 I think we should instead always check if `exe_dir.join(format!("../lib/{}", package_info.package_name()))` exists and if it does, use it as resource path

Reply:
> we should instead always check if `exe_dir.join(format!("../lib/{}", package_info.package_name()))` exists

#11429